api_types.ts

```diff
@@ -775,6 +776,11 @@ export interface ExecutionContext {
    * Information about state of the current sync. Only populated if this is a sync table formula.
    */
   readonly sync?: Sync;
+
+  /**
+   * {@link core.MetadataContext}
+   */
+  readonly parameters?: MetadataContext;
```

Comment:
I've moved towards referring to these as "arguments" instead of "parameters" in my sample code, which I think is more semantically correct. So I probably prefer `arguments` here instead.

Reply:
I was talking to @patrick-codaio on Friday, and he pointed out that even though the docs refer to these as arguments, in the code they are still called parameters. Maybe for the sake of consistency we'd want to keep this as parameters, and then do a whole pass to rename to arguments later?

Reply:
Sounds good to me.